KubeArmor/common/bpffs.go

```diff
@@ -52,23 +54,23 @@ var (
 //
 // We also check and error if there have been multiple mounts at
 // the same point. See - https://patchwork.kernel.org/project/netdevbpf/patch/20220223131833.51991-1-laoar.shao@gmail.com/
-func (be *BPFEnforcer) CheckOrMountBPFFs(bpfRoot string) {
+func CheckOrMountBPFFs(bpfRoot string) {
 	mountOnce.Do(func() {
-		if err := be.checkOrMountBPFFs(bpfRoot); err != nil {
-			be.Logger.Err("Unable to mount BPF filesystem")
+		if err := checkOrMountBPFFs(bpfRoot); err != nil {
+			kg.Err("Unable to mount BPF filesystem")
 		}
```

Comment:
return

Reply:
No executions after that do I don't think it's needed